projects/igniteui-angular/src/lib/core/styles/themes/schemas/light/_snackbar.scss

```diff
@@ -64,3 +64,23 @@ $_bootstrap-snackbar: extend(
     )
 );
 
+/// Generates an indigo snackbar schema.
+/// @type {Map}
+/// @prop {Number} border-radius [2px] - The border radius fraction, between 0-1 to be used for the snackbar component.
```

Comment:
This should be **6px** and the description shouldn't say anything about the fraction.